test/nodetool/test_status.py

```diff
@@ -159,7 +159,14 @@ def _get_ownership(nodes):
     return ownership
 
 
-def _do_test_status(nodetool, keyspace, node_list, resolve=None):
+def _do_test_status(nodetool, keyspace_desc, node_list, resolve=None, table=None):
+    if keyspace_desc:
+        keyspace = keyspace_desc["keyspace"]
+        keyspace_uses_tablets = keyspace_desc["uses_tablets"]
```

Comment:
Use a `NamedTuple` instead.

Reply:
(you can also add the `table` field to it, and remove the separate param)

Reply:
Done via:
`StatusQueryTarget = namedtuple('StatusQueryTarget', ['keyspace', 'table', 'uses_tablets'])`

Reply:
A mode modern version is:
```python
from typing import NamedTuple

class StatusQueryTarget(NameTuple):
    keyspace: str
    table: str
    uses_tablets: bool
```
Notice, how this format allows you to also use type-hints.